collector/filesystem_bsd.go

```diff
@@ -33,6 +33,7 @@ import "C"
 
 const (
 	defIgnoredMountPoints = "^/(dev)($|/)"
+	defIgnoredFsTypes     = "^devfs$"
```

Comment:
defIgnoredFsTypes -> defIgnoredFSTypes (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Similar for the other "Fs" mentions.